### PR TITLE
Add presence to Graph v1.0

### PR DESCRIPTION
**Pull request by elvinyang-msft**
Adding presence doc for Graph v1.0 and include throttling limits.

### CODE REVIEW

File: api-reference/v1.0/api/cloudcommunications-getpresencesbyuserid.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+| Permission type | Permissions (from least to most privileged)                  |
+| :-------------- | :----------------------------------------------------------- |
+| Delegated (work or school account)     | Presence.Read.All                         |
+| Delegated (personal Microsoft account) | Not Supported.                         |
+| Application                            | Not Supported.                                  |
+
+> **Note:**
+> * Maximum of 650 users are supported per API request.
+> * The request rate of this API is limited to 1500 API requests per 30 seconds per app/tenant.
```
Comment (satyakonmsft) [inline api-reference/v1.0/api/cloudcommunications-getpresencesbyuserid.md:29]: per application per tenant

**Review by satyakonmsft — APPROVED** [review]

**Review by jsandoval-msft — CHANGES_REQUESTED** [review]
can we also include a quick update to this: https://docs.microsoft.com/en-us/graph/throttling#cloud-communication-service-limits. We update a patch update earlier, but should adjust to match the 1500 requests per 30 second period mentioned here.

File: api-reference/v1.0/api/presence-get.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+Content-Length: 1574
+
+{
+	"id": "66825e03-7ef5-42da-9069-724602c31f6b",
+	"availability": "DoNotDisturb",
+	"activity": "Presenting"
+}
+```
+
+### Example 3: Get the presence information of another user
```
[the remark this answers is not in the document]
Comment (elvinyang-msft) [reply api-reference/v1.0/api/presence-get.md:161]: That's for subscription

File: api-reference/v1.0/api/cloudcommunications-getpresencesbyuserid.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+If successful, this method returns a `200 OK` response code and a collection of [presence](../resources/presence.md) objects in the response body.
+
+
+## Examples
+
+### Request
+The following example shows a request.
+
+
+# [HTTP](#tab/http)
```
Comment (Lauragra) [inline api-reference/v1.0/api/cloudcommunications-getpresencesbyuserid.md:61]: Please remove this H1.

File: api-reference/v1.0/api/cloudcommunications-getpresencesbyuserid.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+```http
+POST https://graph.microsoft.com/v1.0/communications/getPresencesByUserId
+Content-Type: application/json
+
+{
+	"ids": ["fa8bf3dc-eca7-46b7-bad1-db199b62afc3", "66825e03-7ef5-42da-9069-724602c31f6b"]
+}
+```
+# [C#](#tab/csharp)
```
Comment (Lauragra) [inline api-reference/v1.0/api/cloudcommunications-getpresencesbyuserid.md:75]: Global: We don't bring forward the SDK snippets when we promote APIs to v1.0. Those will get autogenerated via the weekly automation after we merge into master. Please remove the snippet links and snippet files from the PR.

File: concepts/changelog.md [hunk 5]
```diff
@@ -43,6 +43,7 @@ For a summary of the value of these API changes, as well as recent tools, compon
 | Addition | v1.0 | Added the **role** property to the [meetingParticipantInfo](/graph/api/resources/meetingParticipantInfo?view=graph-rest-1.0&preserve-view=true) complex type.|
 | Addition | v1.0 | Added the **lobbyBypassSettings**, **isEntryExitAnnounced**, and **allowedPresenters** property to the [onlineMeeting](/graph/api/resources/onlinemeeting?view=graph-rest-1.0&preserve-view=true) entity to support meeting option configuration.|
 | Addition | beta | Added the **platformId** property to the [participantInfo](/graph/api/resources/participantInfo?view=graph-rest-beta&preserve-view=true) resource. |
+|Addition|v1.0|Added the [presence](/graph/api/resources/presence?view=graph-rest-1.0&preserve-view=true) resource type|
```
Comment (Lauragra) [inline concepts/changelog.md:46]: Please remove this from November and create a new Cloud communications table for December. Also, we need an update to the Skype.Calling.json file in the changelog folder for the new filterable changelog experience.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**